Makefile

```
@@ -1,27 +1,39 @@
 .PHONY: test test-report test-fix update-compatibility-patch
 
 PHP_74_OR_NEWER:=$(shell php -r "echo (int) version_compare(PHP_VERSION, '7.4', '>=');")
+PHP_80_OR_NEWER:=$(shell php -r "echo (int) version_compare(PHP_VERSION, '8.0', '>=');")
```

Needed for my other suggestion:

```suggestion
PHP_VERSION:=$(shell php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;")
PATCH_FILE="tests/php-compatibility-$PHP_VERSION.patch"
```